src/components/TimeCardPage/TimeTableRow.tsx

```diff
@@ -26,6 +35,16 @@ const renderComment = (comment, setComment) => {
 
 } 
 
+const renderCellType = (value:CellType, setValue) => {
```

Comment:
I think this (and the other render methods) could just be their own components, ex. `TimesheetCell`. Besides convention, it'd also be easier to read the code if these are isolated.